src/client/components/sidebar/sidebar.js

```diff
@@ -47,14 +47,19 @@ class Sidebar extends React.Component {
 
         <Leaderboard passedUsers={this.props.G.passed} playerID={this.props.playerID} scores={this.props.G.scores} names={this.props.names} cards={getDealtCardsForPlayers(this.props.G.order, this.props.G.dealt)} />
         {isLastToPass && <div className="warning">You are the last one to pass!</div>}
-        <Button color={(isLastToPass) ? "warning" : "secondary"} className="pass" size="lg" block disabled={
+        {(this.props.ctx.phase === "threats" &&
+          !this.props.G.passed.includes(this.props.playerID) &&
+          this.props.active) && <Button color={(isLastToPass) ? "warning" : "secondary"} className="pass" size="lg" block //disabled={
+
+            //this.props.ctx.phase !== "threats" ||
+            //this.props.G.passed.includes(this.props.playerID) ||
+            //!this.props.active
+
+            //} 
+            onClick={() => { this.props.moves.pass() }}>
```

Comment:
Wouldn't it be enough to write
```js
onClick={this.props.moves.pass}
```
?

Reply:
![needsfunction](https://user-images.githubusercontent.com/86797737/127636937-3ab846eb-6722-46db-9c19-cac8b99a88a8.png)

I think it needs a function. Otherwise I get this error